renderer/components/ActivitiesSection.tsx

```diff
@@ -46,6 +46,7 @@ export default function ActivitiesSection({
           activities={activities}
         />
       </div>
+      <span className="text-xs text-gray-500 pl-6">* - optional field</span>
```

Comment:
Description is also an optional field. Perhaps you should mark "*" and the description, or do not mark any field.